From cb18b1973836de6baf786912af560d1f7086cea9 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Wed, 20 Jul 2016 22:54:33 +0200
Subject: [PATCH] iplink: List valid 'type' argument in ip link help text

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1269528
Upstream Status: iproute2.git commit 771a24289044c

commit 771a24289044c264cfe7b83040c2b35459b4227d
Author: Phil Sutter <phil@nwl.cc>
Date:   Sat Jul 9 11:22:45 2016 +0200

    iplink: List valid 'type' argument in ip link help text

    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 ip/iplink.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/ip/iplink.c b/ip/iplink.c
index 619556c..0098443 100644
--- a/ip/iplink.c
+++ b/ip/iplink.c
@@ -55,7 +55,9 @@ void iplink_usage(void)
 		fprintf(stderr, "                   type TYPE [ ARGS ]\n");
 		fprintf(stderr, "       ip link delete { DEVICE | dev DEVICE | group DEVGROUP } type TYPE [ ARGS ]\n");
 		fprintf(stderr, "\n");
-		fprintf(stderr, "       ip link set { DEVICE | dev DEVICE | group DEVGROUP } [ { up | down } ]\n");
+		fprintf(stderr, "       ip link set { DEVICE | dev DEVICE | group DEVGROUP }\n");
+		fprintf(stderr, "	                  [ { up | down } ]\n");
+		fprintf(stderr, "	                  [ type TYPE ARGS ]\n");
 	} else
 		fprintf(stderr, "Usage: ip link set DEVICE [ { up | down } ]\n");
 
-- 
1.8.3.1

